gifted_build/src/server/database.js

```diff
@@ -0,0 +1,22 @@
+const knexConstructor = require('knex')
+let knex;
+// Temporary: We can determine the actual method of figuring out which environment (dev or prod),
+// and figuring out how to connect to the DB later.
+// Knex has the option to provide a knexfile, with different configurations for dev vs prod
+if (process.env.NODE_ENV == 'production') {
+    knex = knexConstructor({ /* insert production configuration */ })
+} else {
+    knex = knexConstructor({
+        client: 'pg',
+        connection: {
+          user: 'build',
```

Comment:
I don't know if this is a knex thing, but for most postgres installations, the default user is "postgres"

Reply:
Knex uses `pg` underneath, which is the default low-level Postgres driver for Node. I didn't use "postgres" as the user because it has lots of super-user privileges and because I couldn't figure out how to configure the password for it.